wagtail/project_template/project_name/settings.py

```diff
@@ -137,30 +121,12 @@
 )
 
 
-# Template configuration
+# Template context processors
 
 from django.conf import global_settings
-
 TEMPLATE_CONTEXT_PROCESSORS = global_settings.TEMPLATE_CONTEXT_PROCESSORS + (
     'django.core.context_processors.request',
 )
```

Comment:
This will not work with Django 1.9. Since django/django@9ec8aa5e5d42ac4529846f7eae6bf4982800abff, all settings are lists by default. So we could anticipate and use lists on this settings and the others above like `INSTALLED_APPS`, and in this precise case convert the default setting to a list in order make sure we have the right type.

Or the whole file can wait for Django 1.9 compatibility for this change.